src/surface-components/Card/Card.stories.tsx

```diff
@@ -0,0 +1,61 @@
+import { Card, CardImage, CardContent } from "./Card";
+import * as React from "react";
+import { Button } from "../../inputs";
+import imageUrl from "../../assets/card/card.png";
+import PropTypes, { InferProps } from "prop-types";
+import { Meta } from "@storybook/react";
+
+export default {
+  component: Card,
+} as Meta;
+
+interface CardStoryProps {
+  fullWidth?: boolean;
+}
+
+export const Default: React.FC<InferProps<CardStoryProps>> = ({
+  fullWidth,
+}) => {
+  const handlePrimaryButtonClick = () => {
```

Comment:
You should not create the onClick function in the stories outside of the Button. Move the console.log inside the Button.

Reply:
Same as with labels... weird.